servers/Azure.Mcp.Server/vscode/publish-azmcp.ps1

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env pwsh
+
+Write-Host "Publishing Azure MCP Server..." -ForegroundColor Cyan
+
+try {
+    # Resolve and validate project path
+    $projectPath = Join-Path $PSScriptRoot "../src"
+    if (!(Test-Path $projectPath)) {
+        throw "Project path not found: $projectPath"
+    }
+    $projectPath = Resolve-Path $projectPath
+    Write-Host "Project path: $projectPath" -ForegroundColor Gray
+
+    # Setup destination directory
+    $dstBase = Join-Path $PSScriptRoot "server"
+    Write-Host "Output path: $dstBase" -ForegroundColor Gray
+
+    if (!(Test-Path $dstBase)) {
+        Write-Host "Creating output directory..." -ForegroundColor Gray
+        New-Item -ItemType Directory -Path $dstBase -Force | Out-Null
+    }
+
+    # Run dotnet publish with explicit error handling
+    Write-Host "Running dotnet publish..." -ForegroundColor Yellow
+    $output = dotnet publish $projectPath -c Release --self-contained true -o $dstBase 2>&1
```

Comment:
with self contained, I believe specifying a runtime is recommended to avoid errors or framework dependent output.
Can it be injected based on host machine ?